internal/pkg/consensus/expected.go

```diff
@@ -241,10 +212,9 @@ func (c *Expected) validateMining(
 			}
 		}
 
-		// Verify PoStRandomness
-		nullBlkCount := blk.Height - prevHeight - 1
-		if err := c.VerifyPoStRandomness(blk.EPoStInfo.PoStRandomness, electionTicket, workerAddr, uint64(nullBlkCount)); err != nil {
-			return errors.Wrap(err, "PoStRandomness invalid")
+		// Verify EPoSt VRF proof ("PoSt randomness")
+		if err := c.VerifyEPoStVrfProof(ctx, blk.Parents, blk.Height, blk.Miner, workerAddr, blk.EPoStInfo.PoStRandomness); err != nil {
```

Comment:
nit: it would be great to change the name of the postrandomness field to epostvrfproof (maybe keeping the json tag to match spec until changes there merge)

Reply:
Will follow up